src/xcode/ENA/ENA/Source/Services/CCLService/CCLService.swift

```diff
@@ -65,7 +46,24 @@ class CLLService: CCLServable {
 	}
 	
 	func dccWalletInfo(for certificates: [DCCWalletCertificate]) -> Swift.Result<DCCWalletInfo, DCCWalletInfoAccessError> {
-		return .success(DCCWalletInfo())
+		guard let language = Locale.current.languageCodeIfSupported else {
+			return .failure(.unsuppportedLanguage)
```

Comment:
I would argue that since more then 90% of our user base is in Germany the fallback should be German.

Reply:
But the fallback is for people that have their device set to a language that we don't support, and we show the rest of the app in English in that case. So the texts returned in the wallet info should be in English as well

Reply:
From the spec: "If there is no property for the language code, en shall be used. If en is not available, de shall be used."

Reply:
Done

Reply:
But that sentence is for reading the text from the result. The [tech spec for input parameters](https://github.com/corona-warn-app/cwa-app-tech-spec/blob/proposal/logic-engine/docs/spec/dcc-ccl.gen.md#creating-default-ccl-input-parameters) does not specify a fallback, as I think it expects us to only pass in supported languages. The end result might be the same, but the check if "en" is available will never fail, because we know that our app supports "en".

In the context of reading the string from the DCCWalletInfo result it makes more sense to check if "en" is available as we just have a dictionary of languages there. I think for this use case it is not needed

Reply:
After getting feedback from Max, I removed the "contains" check, because CCL will handle unsupported languages. I sill need to resolve the optional to "en". But I don't know wether this is even possible to be nil.